frontend/components/ClubEditPage/MembersCard.tsx

```diff
@@ -85,6 +85,30 @@ export default function MembersCard({ club }: MembersCardProps): ReactElement {
             name: 'email',
           },
         ]}
+        filterOptions={[
+          {
+            label: 'role',
+            options: ['Owner', 'Member', 'Officer'],
```

Comment:
It would be better if you made `options` a list of dictionaries with the keys `id` and `label`. This would make `func` cleaner. Also rename `func` to be more descriptive.

Think of a case where you would render react for the option label. Would you be able to easily compare which option is the correct one?